infrastructure/gke/variables.tf

```diff
@@ -2,3 +2,4 @@ variable "project" {}
 variable "region" {}
 variable "zone" {}
 variable "credentials" {}
+variable cluster_name {}
```

Comment:
I'd stick with using quotes since that's the style in the rest of the TF files

Reply:
done